Emby.ApiClient/model/dto/BaseItemDto.swift

```diff
@@ -372,6 +372,115 @@ public class BaseItemDto : JSONSerializable {
     public var currentProgram: BaseItemDto?
     
     public required init?(jSON: JSON_Object) {
-        fatalError("init(jSON:) has not been implemented: \(jSON)")
+        if  let id = jSON["Id"] as? String {
+            self.id = id
+            
+            // MOVIE START
+            self.name = jSON["Name"] as? String
```

Comment:
I'm not a Swift expert by any means so I was following a pattern that @kevinup7 put together in some earlier work for consistency. I took a look around and found libraries like SwiftyJSON but they still fall back to a model like you see above. 

There is one other library that I found called EVReflection (https://github.com/evermeer/EVReflection), but I've never used it and was hesitant to incorporate it because we'd have to use classes for everything that we'd want to deserialize to and it requires we extend from either `NSObject` or `EVObject` class.